package.json

```diff
@@ -19,7 +19,7 @@
     "lint": "npm-run-all --parallel lint:*",
     "postinstall": "lerna bootstrap",
     "pretest": "npm run lint && node scripts/check-imports.js",
-    "test": "npm run test:unit && npm run test:closure && npm run test:dependency",
+    "test": "npm run test:unit && npm run test:closure && npm run test:dependency && npm run build && npm run clean",
```

Comment:
What about `&& npm run build; npm run clean` so that it always runs `clean`, even if `build` fails?